interactive/settings.py

```diff
@@ -48,6 +48,7 @@
     "interactive",
     "staff",
     "anymail",
+    "debug_toolbar",
```

Comment:
Thanks for adding this and the browser reload. I kept forgetting that they hadn't been added